### ISSUE TITLE
Not a Question, providing info on application compatibility for the Wiki

### ISSUE THREAD

**Issue by elbadcode** [mbucchia/VirtualDesktop-OpenXR#49]
Hello, I have an entry you may wish to add to the Application Compatibility page in the wiki. I found that Blade & Sorcery is compatible if and only if both the OVRPlugin.dll is replaced with the one you provided and the VD json in appdata is modified to specify openxr for this game. The game must then be set to launch in oculus mode. Otherwise steamvr will always initialize first. To simplify this task I have created a [python based patcher with a standalone binary file for users.](https://github.com/elbadcode/BAS-VDXR-patcher/releases/tag/2.0). Please feel free to link to it if this seems suitable. It can also be used as an example if other games require similar fixes.

**Comment by mbucchia** [comment]
Thanks, I've been reading about this and there is a section about OVRPlugin swap on that page, however it's not "supported" and it's a use-at-own-expense thing. The thing with these methods is that the game developer never tested it, so there can be bugs in the game and I don't want to start having people ask me to fix them (not related to VDXR).

**Comment by elbadcode** [comment]
I will say that having tested extensively, in this case it seems to work perfectly and functions identically to the native oculus runtime option on oculus headsets, but I understand why you'd rather not open that can of worms.

Perhaps I should instead post this in the tools section in the VD discord to make it easier to find, but that way it wouldn't come from an official source and lead people to bother you for support? 

Of course the best case scenario is for the developers to add support for a generic openxr loader, but when I petitioned them about it they were too focused on the game's 1.0 release to consider it. This is one of the most drastic improvements to performance/tracking with VDXR I've tested so perhaps I'll give them another reminder.
